breaking-front/src/pages/PostWrite/PostWrite.js

```diff
@@ -4,30 +4,94 @@ import PostUploadMediaForm from 'pages/PostWrite/units/PostWriteUploadMediaForm'
 import 'react-datepicker/dist/react-datepicker.css';
 import dayjs from 'dayjs';
 import PostWriteSearchLocation from 'pages/PostWrite/units/PostWriteSearchLocation';
+import useInputs from 'hooks/useInputs';
+import { useMutation } from 'react-query';
+import { postPostWrite } from 'api/postWrite';
+import MESSAGE from 'constants/message';
+import { useNavigate } from 'react-router-dom';
+import { PAGE_PATH } from 'constants/path';
 
 const PostWrite = () => {
-  const [occurDate, setOccurDate] = useState(dayjs().format('YYYY-MM-DD'));
-  const [occurTime, setOccurTime] = useState(dayjs().format('HH:mm'));
-  const [selectedPostType, setSelectedPostType] = useState('charged');
-  const [selectedAnonymous, setSelectedAnonymous] = useState('public');
-  const [price, setPrice] = useState(0);
+  const navigate = useNavigate();
   const [isShowPriceInput, setIsShowPriceInput] = useState(false);
+  const [mediaList, setMediaList] = useState([]);
+  const [postWriteData, onChangePostWriteData, setPostWriteData] = useInputs({
+    location: undefined,
+    eventTime: dayjs().format('YYYY-MM-DDTHH:mm'),
+    title: '',
+    content: '',
+    price: 0,
+    postType: 'charged',
+    isAnonymous: false,
+    thumbnailIndex: 0,
+  });
 
-  const handlePostType = (event) => {
-    setSelectedPostType(event.target.id);
-  };
-  const handleAnonymous = (event) => {
-    setSelectedAnonymous(event.target.id);
+  const { mutate: PostWriteMutate } = useMutation(postPostWrite, {
+    onSuccess: () => {
+      alert('작성되었습니다.');
+      navigate(PAGE_PATH.HOME);
+    },
+    onError: (error) => {
+      console.log(error);
+      //에러처리
+    },
+  });
+
+  const postWriteSubmit = (event) => {
+    event.preventDefault();
+
+    if (!postWriteData.location) {
+      alert(MESSAGE.POST_WRITE.LOCATION_BLANK);
+      return;
+    } else if (postWriteData.title === '') {
+      alert(MESSAGE.POST_WRITE.TITLE_BLANK);
+      return;
+    } else if (postWriteData.content === '') {
+      alert(MESSAGE.POST_WRITE.CONTENT_BLANK);
+      return;
+    }
+
+    const formData = new FormData();
+
+    let hashtagList = [];
+    postWriteData.content
+      .split(/(#[^\s#]+|\n)/g)
+      .map(
+        (contentSlice) =>
+          contentSlice[0] === '#' && hashtagList.push(contentSlice)
+      );
+    // 해시태그 추출
+
+    formData.append('mediaList', mediaList);
+    formData.append(
+      'data',
+      JSON.stringify({
+        ...postWriteData,
+        hashtagList: hashtagList.length === 0 ? 'null' : hashtagList,
+        price: postWriteData.postType === 'free' ? 0 : postWriteData.price,
+        eventTime: dayjs(postWriteData.eventTime).format('YYYY-MM-DD HH:ss:ss'),
```

Comment:
분 단위를 정확하게 보내지 않는 문제가 있습니다!
```suggestion
        eventTime: dayjs(postWriteData.eventTime).format('YYYY-MM-DD HH:mm:ss'),
```